Lib/ctypes/test/test_loading.py

```diff
@@ -116,6 +116,19 @@ def test_1703286_B(self):
         # This is the real test: call the function via 'call_function'
         self.assertEqual(0, call_function(proc, (None,)))
 
+    @unittest.skipUnless(os.name == "nt",
+                         'test specific to Windows')
+    def test_load_dll_get_attr(self):
+        # This test case verifies that
+        # ctypes.LoadLibrary.__getattr__ raises AttributeError
+        # when name cannot be resolved
+        excepted = False
+        try:
+            hasattr(windll, 'test')
+        except OSError:
+            excepted = True
+        self.assertFalse(excepted)
```

Comment:
Wondering why we need to catch OSError. Isn't that the point of the patch? 😉 

```suggestion
        self.assertFalse(hasattr(windll, 'test'))
```